server/server.js

```diff
@@ -0,0 +1,58 @@
+//**************************** SERVER *****************************
+const express = require('express');
+const server = express();
+const path = require('path');
+const port = process.env.PORT || 3010;
+
+const initialize = () => {
+    router.serveClient();
+    router.allowImageRetrieval();
+    server.listen(port, () => {
+        console.log(`listening on port ${port}`);
+    });
+}
+
+const router = {
+
+    serveClient: () => {
+        server.use(express.static('../client/dist'));
+        server.get('/', (req, res) => {
+            let index = path.join(__dirname, '../client/index.html');
+            res.sendFile(index, (err) => {
+                if (err) throw err;
+                res.end();
+            });
+        });
+    },
+
+    allowImageRetrieval: () => {
+        server.get('/images/:productID/', (req, res) => {
+            let productID = req.params.productID;
+            queryDatabase(productID)
+            .then(document => {res.status(200).send(document).end()})
+            .catch(err => {throw err});
```

Comment:
do you think you need to send back a 404 status code if there's an error or does it do it automatically for you

Reply:
Yes, I need to handle failure status codes, thanks for the reminder. I'll make some changes to that regard in my next version